### PR TITLE
Moves runtime macro out of experimental flag

### PR DESCRIPTION
**Pull request by gupnik**
Step in https://github.com/paritytech/polkadot-sdk/issues/3688

Now that the `runtime` macro (Construct Runtime V2) has been successfully deployed on Westend, this PR moves it out of the experimental feature flag  and makes it generally available for runtime devs.

### CODE REVIEW

File: prdoc/pr_4249.prdoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    description: |
+      Now that the runtime macro (Construct Runtime V2) has been successfully deployed on Westend, 
+      this PR moves it out of the experimental feature flag and makes it generally available for
+      runtime devs.
+
+crates:
+  - name: frame-support
+    bump: patch
+  - name: frame-support-procedural
+    bump: patch
```
Comment (bkchr) [inline prdoc/pr_4249.prdoc:17]:
```suggestion
    bump: minor
  - name: frame-support-procedural
    bump: minor
```

**Review by bkchr — COMMENTED** [review]
Generally not sure if just because it was deployed on one network, it should be taken out of the experimental flag? Given this argument, we would have never needed to add it there in the first place, because no downstream user had even the time to try this out.

**Comment by gupnik** [comment]
> Generally not sure if just because it was deployed on one network, it should be taken out of the experimental flag? Given this argument, we would have never needed to add it there in the first place, because no downstream user had even the time to try this out.

Yeah, makes sense. @ggwpez had suggested to put in under the experimental flag to make sure that the metadata is not impacted before and after the change. I felt that this has been validated now but happy to wait in case you feel something else is needed first.

**Review by kianenigma — COMMENTED** [review]
Which test networks are using this now? 

I have less of a strong opinion on letting it remain experimental, probably would opt for the safer option of letting it be there for now.

Instead, I would put the focus on making sure this is used as much as possible within our codebase/templates.

**Comment by gupnik** [comment]
> Which test networks are using this now?

Currently on westend.

> Instead, I would put the focus on making sure this is used as much as possible within our codebase/templates.

Yeah, my hope was to add it in all test runtimes, but the experimental flag makes it difficult.

**Review by kianenigma — APPROVED** [review]
I agree that the `feature = experimental` has very limited use in this context, maybe we should not use it at all. But that is OT and this should not remain blocked.

**Pull request merged**